app/controllers/foreman_plugin_template/react_controller.rb

```diff
@@ -0,0 +1,24 @@
+module ForemanPluginTemplate
+  class ReactController < ::ApplicationController
```

Comment:
This controller, paths and templates are not needed now, right?
We should point to the core react controller and we should use the global router instead, right?